src/AbstractStatement.php

```diff
@@ -27,7 +27,7 @@ public function __construct(PDO $dbh)
     /**
      * @throws DatabaseException
      *
-     * @return PDOStatement
+     * @return int|string|PDOStatement
```

Comment:
I am not sure what to do with this annotation as function overloading in PHP has gotten more complicated in 7.